zerocopy-derive/tests/ui-nightly/enum.rs

```diff
@@ -114,6 +122,21 @@ enum FromZeros5 {
     B,
 }
 
+struct NotFromZeros;
+
+#[derive(FromZeros)]
+#[repr(u8)]
+enum FromZeros6 {
+    A(NotFromZeros),
+}
```

Comment:
Same as above - needs to be in a separate file.

Reply:
Done